openghg/store/_metstore.py

```diff
@@ -94,7 +115,7 @@ def search(
         from openghg.store.base import Datasource
 
         datasources = (Datasource.load(uuid=uuid, shallow=True) for uuid in self._datasource_uuids)
-
+        #print(self._datasource_uuids)
         # We should only get one datasource here currently
```

Comment:
The search function should be removed and any searching passed to a `search_met` function.